site/docs/spec.md

```diff
@@ -357,6 +357,7 @@ A manifest file must store the partition spec and other metadata as properties i
 | _required_ | _required_ | `partition-spec`    | JSON fields representation of the partition spec used to write the manifest  |
 | _optional_ | _required_ | `partition-spec-id` | ID of the partition spec used to write the manifest as a string              |
 | _optional_ | _required_ | `format-version`    | Table format version number of the manifest as a string                      |
+| _optional_ | _optional_ | `object-type`       | Type of object this metadata file is for: "table" or "view". Default is "table". |
```

Comment:
I would state that "if missing, assumed to be "table"` rather than "default is table". Using "default" doesn't quite seem correct to me.

Reply:
Do not like `object-type` either. How about `manifest-type` over `spec-type`? 

```
Type of this manifest file, if missing, assumed to be "table".
```

Reply:
We need to be careful about using the term "manifest". That refers to manifest files in the table spec and not these metadata files.

I'm okay with "spec-type" or "object-type". I don't think we should worry too much about this.